mtcpclient/groupofconnections.go

```diff
@@ -46,6 +55,59 @@ func (gc GroupOfConnections) AtLeastOneConnectionInError() bool {
 	return gc.isIn(tcpclient.ConnectionError)
 }
 
-func (gc GroupOfConnections) AtLeastOneConnectionEstablished() bool {
-	return gc.isIn(tcpclient.ConnectionEstablished)
+func (gc GroupOfConnections) AtLeastOneConnectionOK() bool {
+	return gc.isIn(tcpclient.ConnectionEstablished) || gc.isIn(tcpclient.ConnectionClosed)
+}
+
+func appendConnections(previous_connections []tcpclient.Connection, new_connections ...tcpclient.Connection) []tcpclient.Connection {
```

Comment:
again, []tcpclient.Connection are group of connections.. why are not you using the types you are creating?

Reply:
the same reason as above, this `appendConnections` and `getFilteredListByStatus` are focused on get a list of connections from a Group of Connection, not to generate another Group of Connections

Reply:
syntactically speaking, a "list of connection" looks as a type of "group of connections", so i don't understand why we cannot use something we are using to map these concepts. I don't connect you comment with any other comment we have "above". Let's have a chat... i'm not following you

Reply:
btw, here is where you are using "_"

Reply:
👍